x-pack/plugins/apm/public/index.js

```diff
@@ -28,6 +30,25 @@ import { history } from './components/shared/Links/url_helpers';
 
 import { I18nContext } from 'ui/i18n';
 
+// render APM feedback link in global help menu
+chrome.helpExtension.set(domElement => {
```

Comment:
Is this the only API for setting the help menu? I was expecting it to be more strictly defined - and if anything that it would assume React. I guess it's a good thing that it's framework agnostic - just not how I expected it to be...